src/Module/Api/Mastodon/Statuses.php

```diff
@@ -77,14 +78,25 @@ public function put(array $request = [])
 			$item['language'] = json_encode([$request['language'] => 1]);
 		}
 
-		if (!empty($request['spoiler_text'])) {
-			if (($request['in_reply_to_id'] == $post['uri-id']) && DI::pConfig()->get($uid, 'system', 'api_spoiler_title', true)) {
-				$item['title'] = $request['spoiler_text'];
+		if ($post['gravity'] == 0) {
```

Comment:
Please use the corresponding constant reference instead of the static value:
```suggestion
		if ($post['gravity'] == Item::GRAVITY_PARENT) {
```